akka-actor/src/main/resources/reference.conf

```diff
@@ -1056,11 +1056,26 @@ akka {
     # immediately when the last phase is reached.
     exit-jvm = off
 
+    # Exit status to use on (System.exit(int)) when 'exit-jvm' is 'on'.
+    exit-code = 0
+
     # Run the coordinated shutdown when the JVM process exits, e.g.
     # via kill SIGTERM signal (SIGINT ctrl-c doesn't work).
     # This property is related to `akka.jvm-shutdown-hooks` above.
     run-by-jvm-shutdown-hook = on
 
+    # When Coordinated Shutdown is triggered an instance of `Reason` is
+    # required. That value can be used to override the default settings.
+    # Only 'exit-jvm', 'exit-code' and 'terminate-actor-system' may be
+    # overriden depending on the reason.
+    reason-overrides {
+      "akka.actor.CoordinatedShutdown.ClusterDowningReason$" {
+        exit-code = -1
+        exit-jvm = on
+        terminate-actor-system = on
```

Comment:
I think we should leave the defaults for `exit-jvm` and `terminate-actor-system` as it was. Would prefer to have the defaults easy to understand, and this would also not be backwards compatible for a patch drop in release (2.5.x).

Reply:
Changing the exit code is fine.